### PR TITLE
The Karters 2: Turbo Charged support

### PR DESCRIPTION
**Pull request by PixelEdgeGames**
The Karters 2: Turbo Charged

If you are adding or modifying an Auto Splitter, please fill out this checklist:
- [Yes] My Auto Splitter does not contain any malicious functionality and I'm entirely responsible for any damages myself. (Any kind of abuse in an Auto Splitter will result in an immediate ban.)
- [Yes] I am not replacing an existing Auto Splitter by a different author, or I have received permission from the author to replace the existing Auto Splitter.
- [Yes] The Auto Splitter has an open source license that allows anyone to fork and host it.

### CODE REVIEW

**Comment by just-ero** [comment]
This auto splitter uses `UnityASL.bin`, which is deprecated. I will PR the auto splitter and report back.

**Comment by PixelEdgeGames** [comment]
Hope it can be approved because we are running demo during Steam Next Fest and we want to make Speedrunning Contest. Thank you

**Comment by just-ero** [comment]
PR opened at https://github.com/PixelEdgeGames/TheKarters2_AutoSplitter/pull/1.

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -18574,4 +18574,16 @@
 	<Type>Script</Type>
 	<Description>Autosplitter and Loadless timer made by WillTreaty</Description>
     </AutoSplitter>
+	<AutoSplitter>
+        <Games>
+            <Game>The Karters 2: Turbo Charged</Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/PixelEdgeGames/TheKarters2_AutoSplitter/main/TheKarters2.asl</URL>
+            <URL>https://github.com/just-ero/asl-help/raw/main/libraries/UnityASL.bin</URL>
```
Comment (just-ero) [inline LiveSplit.AutoSplitters.xml:18583]:
```suggestion
            <URL>https://github.com/just-ero/asl-help/raw/main/lib/asl-help</URL>
```

**Comment by PixelEdgeGames** [comment]
Thank you for your help!

**Comment by PixelEdgeGames** [comment]
Hello @just-ero - do we need to do something extra to game appear in Games list? I am refreshing since yesturday and unfortunately no luck. The game is The Karters 2: Turbo Charged

**Comment by just-ero** [comment]
This list has nothing to do with the games list that appears in the splits editor. That list is pulled from the speedrun.com entries.

**Comment by just-ero** [comment]
You can simply enter the game name manually already and the auto splitter will become activatable.

To make it appear in the auto-complete list, the game must be on speedrun.com, yes.

**Pull request merged**